modules/virt-about-hostpath-provisioner.adoc

```diff
@@ -1,28 +1,11 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc
```

Comment:
Module metadata [guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)

Reply:
I looked at the guidelines, but I'm not sure about how to proceed. I thought that I had found all xref instances and fixed them, but now I'm seeing this for each of the virt folders:

![image](https://user-images.githubusercontent.com/45106934/226463181-dd87888b-7d2c-40eb-8f42-a9678df516af.png)

Reply:
Done! Thanks